src/components/ProgressReportListBreadcrumb/ProgressReportListBreadcrumb.tsx

```diff
@@ -0,0 +1,22 @@
+import { Skeleton } from '@mui/material';
+import { IdFragment } from '~/common';
+import { Breadcrumb, BreadcrumbProps } from '../Breadcrumb';
+import { idForUrl } from '../Changeset';
+
+interface ProgressReportListBreadcrumbProps extends Partial<BreadcrumbProps> {
+  data?: IdFragment;
```

Comment:
I'd rename this for clarity.
```suggestion
  engagement?: IdFragment;
```